src/server/models/list.ts

```diff
@@ -236,6 +236,28 @@ export async function removeUserFromList(listId: number, userEmail: string): Pro
   }
 }
 
+export async function deleteUserByEmail(email: string, adminEmail: string): Promise<void> {
```

Comment:
just noticed the file - this should be in `models/user.ts`, with `createUsersFromEmail` and `checkUserExists`